client/app/main/student/partial/interventions.html

```diff
@@ -15,11 +15,9 @@
         <dt>Action Taken</dt>
         <dd>
           <div class="input-group">
-            <span class="input-group-btn">
-              <button type="button" class="btn btn-success" ng-click="open($index)">
-                <i class="fa fa-calendar"></i>
-              </button>
-            </span>
+            <input type="text" class="form-control truncate" readonly value="{{note.note}}"/>
+            <span class="input-group-addon">{{note.date | date:'yyyy-MM-dd'}}</span>
```

Comment:
This doesn't look like it is merged correctly. You replace the action taken section with the notes section.